Shell/Naming.cpp

```diff
@@ -1106,42 +1108,67 @@ bool Naming::canBeInDefinition(Formula* f, Where where) {
   return true;
 }
 
-Literal* Naming::getDefinitionLiteral(Formula* f, Formula::VarList* freeVars) {
+Literal* Naming::getDefinitionLiteral(Formula* f, VList* freeVars) {
   CALL("Naming::getDefinitionLiteral");
 
-  unsigned length = Formula::VarList::length(freeVars);
-  unsigned pred = env.signature->addNamePredicate(length);
-  Signature::Symbol* predSym = env.signature->getPredicate(pred);
-
-  if (env.colorUsed) {
+  unsigned length = VList::length(freeVars);
+  /*if (env.colorUsed) { //TODO what are colors all about?
     Color fc = f->getColor();
     if (fc != COLOR_TRANSPARENT) {
       predSym->addColor(fc);
     }
     if (f->getSkip()) {
       predSym->markSkip();
     }
```

Comment:
This looks like we are breaking so color-handling code? 

Could you revive if in the non-appify branch?

Reply:
This has been revived.